spec/plugin/init_spec.rb

```diff
@@ -35,52 +35,39 @@
   end
 
   context 'init detection (real)' do
-    context 'with ubuntu:18.04' do
+    context 'with ubuntu:24.04' do
       skip 'finds systemd' do
```

Comment:
The skip is inherited from https://github.com/xing/fpm-fry/pull/17, so I don't really get the exact reasoning for this. But looks like our internal ubuntu images have systemd installed while upstream ubuntu does not.